src/test/kotlin/no/nav/syfo/application/narmesteleder/api/NarmestelederTestUtil.kt

```diff
@@ -0,0 +1,78 @@
+package no.nav.syfo.application.narmesteleder.api
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import no.nav.syfo.narmestelederrelasjon.kafka.NARMESTE_LEDER_RELASJON_TOPIC
+import no.nav.syfo.narmestelederrelasjon.kafka.domain.NY_LEDER
+import no.nav.syfo.util.configuredJacksonMapper
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.apache.kafka.clients.consumer.ConsumerRecords
+import org.apache.kafka.common.TopicPartition
+import testhelper.UserConstants
+import testhelper.UserConstants.ARBEIDSTAKER_FNR
+import testhelper.generator.generateNarmesteLederLeesah
+import testhelper.mock.toHistoricalPersonIdentNumber
+import java.time.OffsetDateTime
+
+fun generateNarmestelederTestdata(): ConsumerRecords<String, String> {
+    val objectMapper: ObjectMapper = configuredJacksonMapper()
+
+    val partition = 0
+    val narmesteLederRelasjonTopicPartition = TopicPartition(
+        NARMESTE_LEDER_RELASJON_TOPIC,
+        partition,
+    )
+    val narmesteLederLeesah = generateNarmesteLederLeesah(
```

Comment:
Hva er Leesah? 😅

Reply:
Godt spørsmål - jeg har bare skyflet rundt på noe kode, så jeg vet ikke. Det er noe Kafka-relatert. Vi får høre med Erik.

Reply:
Det står for `Livet Er En Strøm Av Hendelser`, fordi hendelser skal strømme på kafka 👍